package.json

```diff
@@ -34,11 +34,11 @@
     "connect-redis": "^3.3.0",
     "express": "^4.16.2",
     "express-session": "^1.15.3",
+    "handy-redis": "^1.3.0",
```

Comment:
What does this thing give us?

Reply:
why dont just promisify redis get/sets instead depending on a 3rd party module which is could not be maintained in the future?

Example:
```
const client = redis.createClient();
const util = require('util');
const asyncGet = util.promisify(client.get).bind(client);
```

Reply:
@AWaselnuk  

`handy-redis` is basically just doing the same thing Ivan is suggesting internally.

I don't have super strong feelings about it so I'll probably switch it to a manually promisified implementation.